packages/block-library/src/block/test/fixture.js

```diff
@@ -0,0 +1,24 @@
+export const getReusableBlock = ( id ) => ( {
```

Comment:
Would it be beneficial to rename this to be explicit about its purpose and return? 
```suggestion
export const getMockedReusableBlock = ( id ) => ( {
```

Reply:
Yeah, good point, I'll rename it as you pointed, thanks!

Reply:
I applied your suggestion in this [commit](https://github.com/WordPress/gutenberg/pull/32038/commits/e99d7cf2a7c577596171f77b6c658ea6dd2581d8).